### ISSUE TITLE
Debugmsg when NPC uses a rag to stop bleeding

### ISSUE THREAD

**Issue by sethsimon** [CleverRaven/Cataclysm-DDA#20208]
The rag properly turns into a blood soaked rag, and it properly has a chance to stop the bleeding, so the only issue here is the debug message, which says:
```
Tried to consume charges for non-tool, non-food, non-med item
FUNCTION: bool player::consume_charges(item&, long int)
```

**Comment by kolsurma** [comment]
Okay so the issue is that it turns into a blood soaked rag as soon as "charges_used" in the method npc::heal_self() is assigned, thus making it no longer a medical item once it reaches the debug of player::consume_charges.

I'm unsure whether I should add a specific check to the debug or to check if it's a rag before it calls consume_charges, as it still applies the item correctly.

This may interact with #20090

**Comment by illi-kun** [comment]
Probably fixed by #20332.
